### ISSUE TITLE
Make subtree tags useable for integrator / editor

### ISSUE THREAD

**Issue by mhsdesign** [neos/neos-development-collection#5380]
Subtask of https://github.com/neos/neos-development-collection/issues/3732 

With https://github.com/neos/neos-development-collection/issues/4550 subtreetags were introduced. But currently there is no way to leverage them without using the core api.

Especially as the tags can be used for security (https://github.com/neos/neos-development-collection/pull/5298) a Neos out of the box solution would be important to make this an actual feature.

One option is to extend NodeType schema by some "default tags" that will be set if a node of that type is created.
Though this opens the discussion again of https://github.com/neos/neos-development-collection/issues/4484 which claims that this should better be done with node templates like `template.tags = 'foo'`.
Basically it is currently the way that only properties (and references [bug](https://github.com/neos/neos-development-collection/issues/4380)) can be created with the node creation and thus default values would make sense. Everything else could be argued - as long as subtree tags cannot be set via `CreateNodeAggregateWithNode` are separate.

Other options would be: provide an editor to edit subtree tags? Or hack it into `options.subtreeTags` and evaluate it only via the Ui.
